sonar-php-plugin/src/main/java/org/sonar/plugins/php/PHPRulesDefinition.java

```diff
@@ -32,7 +42,77 @@
   public void define(Context context) {
     NewRepository repository = context.createRepository(CheckList.REPOSITORY_KEY, Php.KEY).setName(REPOSITORY_NAME);
     new AnnotationBasedRulesDefinition(repository, Php.KEY).addRuleClasses(false, CheckList.getChecks());
+
+    for (NewRule rule : repository.rules()) {
```

Comment:
@vilchik-elena Agreed. I usually attempt to make separate commits, in order to make the review less painful. In this case the removal of the annotations showed that this code was missing.